### ISSUE TITLE
NestedNotOperatorsInspector: !bool [!=]== true|false

### ISSUE THREAD

**Issue by rentalhost** [kalessil/phpinspectionsea#226]
_This is one of issue that I think that exists by some reason, but I can't found it on inspections available. Then, sorry me if it is a dup._

**Check the code below:**

```php
echo !$bool !== true;
echo !$bool === true;
```

The expression result will be reverted by the _not operator_. But you can rewrite it to make more simple, without change behavior:

```php
echo $bool === true;
echo $bool !== true;
```

Basically, if your expression is _not'ed_ and is strictly _(not-)equal_ a boolean value, then this inspection should be triggered.

_Autofixer welcome!_

**Comment by kalessil** [comment]
It doesn't exist, and it is easy to pick this one =)

**Comment by rentalhost** [comment]
As this is slight related to #239, I'll improve this last instead of create a new inspection. But for that, I should wait for review of the PR #249.

**Comment by kalessil** [comment]
Implemented!
